package.json

```diff
@@ -32,6 +32,7 @@
   },
   "scripts": {
     "postinstall": "bash ./bin/install-hooks.sh",
+    "build:frameworks:config": "node build/frameworks.config.js",
```

Comment:
@Jinjiang `build:config` maybe another script for all config about, naming `frameworks:config` is more accurate now.